### PR TITLE
When configuring the kafka client disable the server

### PR DESCRIPTION
**Pull request by jvlcek**
Fixes https://github.com/ManageIQ/manageiq-appliance_console/issues/141

The overhead imposed by running the kafka server is unnecessary when only the kafka client is desired.

The best way to ensure resources used by the kafka server are cleaned up is to invoke the kafka servers `#unconfigure` method. Running the `#unconfigure` method is only necessary if the kafka or zookeeper services are not running. It could be argued that the new method `#configured?` only need check if the kafka service is running but to be safe I check if either kafka or zooker services are running.

### CODE REVIEW

**Comment by jvlcek** [comment]
@miq-bot add_label enhancement

**Comment by jvlcek** [comment]
@miq-bot assign @bdunne

File: spec/message_configuration_server_spec.rb [hunk 1]
```diff
@@ -272,4 +272,27 @@
       subject.send(:restart_services)
     end
   end
+
+  describe "#configured?" do
```
Comment (bdunne) [inline spec/message_configuration_server_spec.rb:276]: Can you add a test case for false?

Comment (jvlcek) [reply spec/message_configuration_server_spec.rb:276]: Thank you @bdunne. That would indeed be a good idea!

Addressed by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/154/commits/13a4c579bc5e970afbd502c5a8194e5a3b10f391

**Pull request merged**